### DISCUSSION TITLE
How this happened

### DISCUSSION THREAD

**Discussion by Maixnert** [lllyasviel/Fooocus#2524] [Q&A] [answered]
![2024-03-13_12-19-13_2936](https://github.com/lllyasviel/Fooocus/assets/109583170/176196f1-31b1-43bc-81ba-a27e1fcec848)
It is outpainted image. I would Like to know what I did wrong 
`{
"prompt": "(((masterpiece))),(((best quality))), space ship cockpit, ,surreal dramatic lighting shadow (lofi, analog),kodak film by Brandon Woelfel Ryan McGinley,vignette,high budget,bokeh,cinemascope,moody,epic,gorgeous,film grain,grainy\nNegative Prompt\t",
"negative_prompt": "",
"prompt_expansion": "",
"styles": "['Fooocus Cinematic']",
"performance": "Quality",
"resolution": "(1024, 1024)",
"guidance_scale": 4,
"sharpness": 2,
"adm_guidance": "(1.5, 0.8, 0.3)",
"base_model": "thinkdiffusionxl_v10.safetensors",
"refiner_model": "None",
"refiner_switch": 0.5,
"sampler": "dpmpp_2m_sde_gpu",
"scheduler": "karras",
"seed": 709160220196259816,
"lora_combined_1": "xl_more_art-full_v1.safetensors : 0.1",
"metadata_scheme": false,
"version": "Fooocus v2.2.1"
}`

**Comment by VoyagerStray** [comment, marked as answer]
Try using outpainting one iteration per side, e.g. first left, next iteration right and so on. I've noticed that if you specify all sides at the same time, the quality is worse. It might also be worth shortening the prompt, leaving only a general description of the scene and removing the styles

**Reply by bblackmoor** [reply]
I have found this, as well. I have also found that it may take the algorithm a dozen or so tries before the outpainted side is something I want to use as a basis for the outpainting next side.
